src/components/molecules/NavButtons/NavButtons.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { Link } from 'react-router'
+
+import { Button } from 'components/atoms/Button/Button'
+
+interface NavButtonsProps {
+  readonly location?: string
+}
+
+export class NavButtons extends React.Component<NavButtonsProps, undefined> {
+  render() {
+    if (![''].includes(this.props.location)) return null
```

Comment:
Can you explain why?

Reply:
To only render the buttons if it is on the landing page.

Reply:
I agree with @lautarodragan :)

Reply:
Ok. I've updated the component. At the time I was thinking that it would be better for the component to decide it's own behavior rather than its parent deciding.